cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -15,35 +15,44 @@ import (
 	"github.com/square/spiffe-aws-assume-role/cmd/spiffe-aws-assume-role/cli/mappers"
 	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
 	"github.com/square/spiffe-aws-assume-role/pkg/processcreds"
+	"github.com/square/spiffe-aws-assume-role/pkg/telemetry"
 )
 
 type CredentialsCmd struct {
-	Audience        string        `required:"" help:"SVID JWT Audience. Must match AWS configuration"`
-	SpiffeID        string        `required:"" help:"The SPIFFE ID of this workload"`
-	WorkloadSocket  string        `optional:"" help:"Path to SPIFFE Workload Socket"`
-	RoleARN         string        `required:"" help:"AWS Role ARN to assume"`
-	SessionName     string        `optional:"" help:"AWS Session Name"`
-	STSEndpoint     string        `optional:"" help:"AWS STS Endpoint"`
-	STSRegion       string        `optional:"" help:"AWS STS Region"`
-	SessionDuration time.Duration `optional:"" type:"iso8601duration" help:"AWS session duration in ISO8601 duration format (e.g. PT5M for five minutes)"`
-	LogFilePath     string        `optional:"" help:"Path to log file"`
+	Audience         string        `required:"" help:"SVID JWT Audience. Must match AWS configuration"`
+	SpiffeID         string        `required:"" help:"The SPIFFE ID of this workload"`
+	WorkloadSocket   string        `optional:"" help:"Path to SPIFFE Workload Socket"`
+	RoleARN          string        `required:"" help:"AWS Role ARN to assume"`
+	SessionName      string        `optional:"" help:"AWS Session Name"`
+	STSEndpoint      string        `optional:"" help:"AWS STS Endpoint"`
+	STSRegion        string        `optional:"" help:"AWS STS Region"`
+	SessionDuration  time.Duration `optional:"" type:"iso8601duration" help:"AWS session duration in ISO8601 duration format (e.g. PT5M for five minutes)"`
+	LogFilePath      string        `optional:"" help:"Path to log file"`
+	TelemetryAddress string        `optional:"" help:"Socket address (TCP/UNIX) to emit metrics to (e.g. 127.0.0.1:8200)"`
 }
```

Comment:
Whitespace churn. TelemetryAddress is the only new parameter.

Also... there's another parameter called WorkloadSocket, perhaps it would make more sense to call the new parameter TelemetrySocket? TelemetryEndpoint?

Reply:
Yeah, for consistency, `TelemetrySocket` might be good IMO.

Reply:
Will do!